### PR TITLE
get single user

### PR DESCRIPTION
**Pull request by ligopsychakis-michalis**
route to get a single user added to routes.js

### CODE REVIEW

File: backend/src/routes.js [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+});
+
+
+//route to get a single user
+router.get('/users/:id', async (req,res) => {
+    try{
+        let user = await Users.findById(req.params.id);
+        if (user){
+            delete user.password;
+            res.status(200).json(user);
```
Comment (zannis) [inline backend/src/routes.js:17]: Please wrap this in an object with a "user" property as per the acceptance criteria.

File: backend/src/routes.js [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
+        }else{
+            res.status(404).send({ error: `The given id ${req.params.id} not found.` });
+        };
+    }catch(err){
+        res.status(404).send({ error: err });
```
Comment (zannis) [inline backend/src/routes.js:22]: Better return statusCode 500 if the handler crashes for a different reason.

File: backend/src/routes.js [hunk 3]
```diff
@@ -1,12 +1,32 @@
-const express = require('express')
+const express = require('express');
+
```
Comment (zannis) [inline backend/src/routes.js:2]: I think this is also missing the import for the Users entity that you can get after merging #16. ;)

File: backend/src/routes.js [hunk 4]
```diff
@@ -24,6 +24,21 @@ router.get('/users/:id', async (req,res) => {
 });
 
 
+//route to get a single user
+router.get('/users/:id', async (req,res) => {
+    try{
+        let user = await Users.findById(req.params.id);
```
Comment (student0b4dc0d3) [inline backend/src/routes.js]: Mike, I don't think this line of code is correct.
"Users" should be undefined here and Users.findById(..) probably will raise an exception.

The Schemas and Models have been modified to singular form, and to access them you should require the Schema.js
An example of this is line 10 of "index.js" and also a comment I have posted on slack - back-end channel.

Would you please run the server and verify the code is both running and the result is as expected?
I may be wrong in my assumption, in which case please forgive me and disregard this comment.

Best regards and have a great day

**Pull request merged**